Code/client/World.h

```diff
@@ -45,6 +46,8 @@ struct World : entt::registry
     static void Create() noexcept;
     [[nodiscard]] static World& Get() noexcept;
 
+    ActorHandle playerHandler;
```

Comment:
This shouldn't be stored in World. Use Actor::GetHandleByFormId() or whatever the in game function is.

Reply:
Using this workaround because I haven't been able to implement Actor::GetHandle() yet. Will be removed when I do.